maths/difference_sort.py

```diff
@@ -0,0 +1,34 @@
+def diff(a=int, c=int) -> int:
+    """
+    Sorting 2 list numbers without same numbers
+    >>> diff(6,6)
+    Enter 1st list numbers: 1 2 3 6 4 7
+    Enter 2nd list numbers: 1 2 36 96 78 99
+    Sorted list:
+    3
+    4
+    6
+    7
+    36
+    78
+    96
+    99
+    """
+    # Taking input lists
+    b = input("Enter 1st list numbers: ").split()
+    d = input("Enter 2nd list numbers: ").split()
+    # Making set of list
```

Comment:
```suggestion
    # Making set of list
```
Algorithms take input as parameters